test/test-server/utils.go

```diff
@@ -18,11 +18,12 @@ import (
 )
 
 var (
-	serviceAccountInvalidErrMsg = `{"errors":[{"status":"403","detail":"service account is not valid"}]}`
-	roleNameInvalidErrMsg       = `{"status_code":400,"message":"Invalid role name : %s","type":"INVALID REQUEST DATA"}`
-	resourceNotFoundErrMsg      = `{"errors":[{"detail":"resource not found"}], "message":"resource not found"}`
-	badRequestErrMsg            = `{"errors":[{"status":"400","detail":"Bad Request"}]}`
-	userConflictErrMsg          = `{"errors":[{"detail":"This user already exists within the Organization"}]}`
+	serviceAccountInvalidErrMsg    = `{"errors":[{"status":"403","detail":"service account is not valid"}]}`
+	roleNameInvalidErrMsg          = `{"status_code":400,"message":"Invalid role name : %s","type":"INVALID REQUEST DATA"}`
+	resourceNotFoundErrMsg         = `{"errors":[{"detail":"resource not found"}], "message":"resource not found"}`
+	badRequestErrMsg               = `{"errors":[{"status":"400","detail":"Bad Request"}]}`
+	userConflictErrMsg             = `{"errors":[{"detail":"This user already exists within the Organization"}]}`
+	feedbackExceedsMaxLengthErrMsg = `{"errors":[{"status":"403","detail":"feedback exceeds the maximum length"}]}`
```

Comment:
Looks like this only gets used in one place, so let's decrease it's scope (it doesn't need to be a global variable)